instance/Land_Unit.xml

```diff
@@ -1166,7 +1166,7 @@
         <DigitTwo>0</DigitTwo>
       </EntityCode>
       <EntityTypes>
-        <EntityType ID="BUREAU_ATF" Label="Bureau of Alcohol Tobacco Firearms and Explosives (ATF) (Department of Justice)" LabelAlias="ATF DOJ" Graphic="10200100.svg">
+        <EntityType ID="BUREAU_ATF" Label="Bureau of Alcohol Tobacco Firearms and Explosives (ATF) (Department of Justice)" LabelAlias="ATF DOJ" Graphic="10200100.svg" Standard="MILSTD_2525">
```

Comment:
This change is an example of using the `Standard` attribute to differentiate between books.